ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+**Summary:** 
+
+Summarize your issue in one sentence (what goes wrong, what did you expect to happen)
+
+**Steps to reproduce:** 
+
+How can we reproduce the issue?
+
+**Add System logs:**
+
+Add logcat files here(If you are able to add).
+
+**Expected behavior:** 
+
+What did you expect the App to do?
+
+**Observed behavior:** 
+
+What did you see instead?  Describe your issue in detail here.
+
+**Device and Android version:** 
+
+What make and model device (e.g., Samsung J7) did you encounter this on?  What Android
+version (e.g., Android 4.0 Ice Cream Sandwich or Android 6.0 Marshmallow) are you running?  Is it
+ the stock
+version from the manufacturer or a custom ROM ?
```

Comment:
App version will also be helpful. Check out this link for more info about how to file a bug report. https://www.mediawiki.org/wiki/How_to_report_a_bug

Reply:
Sir, I had added it. Can we merge it now?